assets/xml/objects/object_oE4.xml

```diff
@@ -0,0 +1,58 @@
+        <DList Name="gOE4DL8" Offset="0x39E0"/>
+        <DList Name="gOE4DL9" Offset="0x3A98"/>
+        <DList Name="gOE4DL10" Offset="0x3BA0"/>
+        <DList Name="gOE4DL11" Offset="0x3D48"/>
+        <DList Name="gOE4DL12" Offset="0x3E38"/>
+        <DList Name="gOE4DL13" Offset="0x3F28"/>
+        <DList Name="gOE4DL14" Offset="0x3FE0"/>
+        <DList Name="gOE4DL15" Offset="0x4098"/>
+        <Limb Name="gOE4Limb1" Type="Standard" Offset="0x0"/>
+        <Limb Name="gOE4Limb2" Type="Standard" Offset="0xC"/>
+        <Limb Name="gOE4Limb3" Type="Standard" Offset="0x18"/>
+        <Limb Name="gOE4Limb4" Type="Standard" Offset="0x24"/>
+        <Limb Name="gOE4Limb5" Type="Standard" Offset="0x30"/>
+        <Limb Name="gOE4Limb6" Type="Standard" Offset="0x3C"/>
+        <Limb Name="gOE4Limb7" Type="Standard" Offset="0x48"/>
+        <Limb Name="gOE4Limb8" Type="Standard" Offset="0x54"/>
+        <Limb Name="gOE4Limb9" Type="Standard" Offset="0x60"/>
+        <Limb Name="gOE4Limb10" Type="Standard" Offset="0x6C"/>
+        <Limb Name="gOE4Limb11" Type="Standard" Offset="0x78"/>
+        <Limb Name="gOE4Limb12" Type="Standard" Offset="0x84"/>
+        <Limb Name="gOE4Limb13" Type="Standard" Offset="0x90"/>
+        <Limb Name="gOE4Limb14" Type="Standard" Offset="0x9C"/>
+        <Limb Name="gOE4Limb15" Type="Standard" Offset="0xA8"/>
+        <Limb Name="gOE4Limb16" Type="Standard" Offset="0xB4"/>
+        <Limb Name="gOE4Limb17" Type="Standard" Offset="0xC0"/>
+        <Limb Name="gOE4Limb18" Type="Standard" Offset="0xCC"/>
+        <Limb Name="gOE4Limb19" Type="Standard" Offset="0xD8"/>
+        <Limb Name="gOE4Limb20" Type="Standard" Offset="0xE4"/>
+        <Limb Name="gOE4Limb21" Type="Standard" Offset="0xF0"/>
+        <Limb Name="gOE4Limb22" Type="Standard" Offset="0xFC"/>
+        <Limb Name="gOE4Limb23" Type="Standard" Offset="0x108"/>
+        <Limb Name="gOE4Limb24" Type="Standard" Offset="0x114"/>
+        <Limb Name="gOE4Limb25" Type="Standard" Offset="0x120"/>
+        <Limb Name="gOE4Limb26" Type="Standard" Offset="0x12C"/>
+        <Limb Name="gOE4Limb27" Type="Standard" Offset="0x138"/>
+        <Limb Name="gOE4Limb28" Type="Standard" Offset="0x144"/>
+        <Limb Name="gOE4Limb29" Type="Standard" Offset="0x150"/>
+        <Limb Name="gOE4Limb30" Type="Standard" Offset="0x15C"/>
+        <Limb Name="gOE4Limb31" Type="Standard" Offset="0x168"/>
+        <Limb Name="gOE4Limb32" Type="Standard" Offset="0x174"/>
+        <Limb Name="gOE4Limb33" Type="Standard" Offset="0x180"/>
+        <Limb Name="gOE4Limb34" Type="Standard" Offset="0x18C"/>
+        <Limb Name="gOE4Limb35" Type="Standard" Offset="0x198"/>
+        <Limb Name="gOE4Limb36" Type="Standard" Offset="0x1A4"/>
+        <Limb Name="gOE4Limb37" Type="Standard" Offset="0x1B0"/>
+        <Limb Name="gOE4Limb38" Type="Standard" Offset="0x1BC"/>
+        <Hierarchy Name="gOE4Skel" Type="Normal" LimbType="Standard" Offset="0x260"/>
```

Comment:
naming the limbs isn't a huge deal right now but maybe do gUnusedRedHairedKokiri instead of gOE4. It's not a big deal if the names are long in the case of objects.